docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "The Fractribution data model"
+date: "2022-12-01"
+sidebar_position: 105
+---
+
+# Snowplow Fractribution Package
+
+
+## Overview
+
+Snowplow Fractribution is a marketing attribution model for dbt. The name itself comes from `Fractional attribution`, which allows you to attribute the value of a conversion to one or more channels depending on the conversion pathway. As a result, it becomes possible to determine the revenue per channel, as well as ROAS once you have cost data for each marketing channel.
+
+This package consists of a series of dbt models with the goal to produce the following models:
+
+- **snowplow_fractribution_channel_counts**: Number of sessions grouped by channel, campaign, source and medium.
+- **snowplow_fractribution_channel_spend**: Spend on each channel, used in ROAS calculations.
+- **snowplow_fractribution_conversions_by_customer_id**: Conversion revenue for each conversion, along with the associated customerid.
+- **snowplow_fractribution_path_summary**: Summary of different path combinations and associated conversion/non-conversions.
+- **snowplow_fractribution_paths_to_conversion**: Path combinations leading to conversion.
+- **snowplow_fractribution_paths_to_non_conversion**: Path combinations leading to non-conversion.
+- **snowplow_fractribution_sessions_by_customer_id**: Channel information by session timestamp, where an event timestamp is considered as the session start.
+
+Once the models are generated, the next step is to run a python script which is included in the package to run the fractribution calculations.
```

Comment:
The Python script is currently not in the dbt package - it's in the marketing accelerator. It's a good call out - is there some way that we can/should add the python script to the package, or just link to the docker image (once created) in the readme/documentation?

Reply:
Thank you for raising this, I think it is better to keep the python file within easy reach, that is in the package itself. We have a new package coming up that utilises python scripts at some point and it is was added to the dbt package under a new folder called 'Utils', I think we can do the same here: 
<img width="280" alt="image" src="https://user-images.githubusercontent.com/95634439/205874023-bf5242ee-77f1-498f-83d1-45e7f7dfb0ec.png"> Once the docker image is ready, we can add it to this doc as well (as well as perhaps in the README section of the model)(?)

Reply:
Oh great, yes I agree, having it in easy reach is best. I'll add the utils folder to the package, and add some more information to the readme.